CGPs/cgp-0054.md

```diff
@@ -0,0 +1,183 @@
+---
+cgp: 17
+title: Reopening of the Celo Community Fund
+date-created: 
+author: @MayaRB#2593 @Wade | TPT#4475 @Thylacine | VladiatorLabs.io#6470
```

Comment:
I'm a member of the Celo git organisation under @aaronmboyd so might be worthwhile adding that also